tests/php/QuickFeedbackExtensionTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+class TestController extends Controller {
+    /**
+     * @var array
+     */
+    public static $allowed_actions = array(
+        'form_appears_and_saves',
+    );
+
+    /**
+     * @return HTMLText
+     */
+    public function form_appears_and_saves() {
+        return $this->renderWith('TestController_form_appears_and_saves');
+    }
+}
+
+/**
+ * @mixin PHPUnit_Framework_TestCase
+ */
+class QuickFeedbackExtensionTest extends SapphireTest {
+    /**
+     * @test
+     */
+    public function form_appears_and_saves() {
+        Config::inst()->update('Controller', 'extensions', ['QuickFeedbackExtension']);
```

Comment:
the short array here []... are we requiring a version of php (5.4+) that supports this in composer.json? Might cause issue with other users relying on the lower supported versions for 3.x core. (I'm ok with this personally just thinking wider)